packages/toolkit/src/constant/pipeline.ts

```diff
@@ -76,6 +76,10 @@ export const generateInputsPayload = (fields: PipelineVariableFieldMap) => {
           ];
           break;
         }
+        case "*/*": {
+          input[key] = "Please put your file base64 encoded string";
+          break;
+        }
```

Comment:
```suggestion
        case "file":
        case "*/*": {
          input[key] = "Please put your file base64 encoded string";
          break;
        }
```

@orangecoloured Please help me add this condition too. Since backend will soon change the case and we can adapt this right now